src/main/resources/graphiql.html

```diff
@@ -37,8 +37,8 @@
       copy them directly into your environment, or perhaps include them in your
       favored resource bundler.
      -->
-    <link rel="stylesheet" href="//cdn.jsdelivr.net/graphiql/0.8.0/graphiql.css" />
-    <script src="//cdn.jsdelivr.net/graphiql/0.8.0/graphiql.min.js"></script>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/graphiql/0.9.3/graphiql.css" />
+    <script src="//cdn.jsdelivr.net/graphiql/0.9.3/graphiql.min.js"></script>
```

Comment:
Does the rest of the HTML need to be updated as well or just the JS/CSS?

As a side note, I'll probably end up embedding these files in the repo instead of relying on a CDN in case I lose Internet connection during the live demo 😅.